src/treeview_sandbox.ml

```diff
@@ -0,0 +1,206 @@
+open Import
+
+module Dependency = struct
+  type t = Sandbox.Package.t
+
+  let t_of_js : Ojs.t -> t = Obj.magic
+
+  let t_to_js : t -> Ojs.t = Obj.magic
+
+  let label t = Sandbox.Package.name t
+
+  let description t = Promise.return (Some (Sandbox.Package.version t))
+
+  let tooltip t = Sandbox.Package.synopsis t
+
+  let context_value t =
+    match Sandbox.Package.documentation t with
+    | Some _ -> "package-with-doc"
+    | None -> "package"
+
+  let icon _ =
+    TreeItem.LightDarkIcon.
+      { light = `String (Node.__filename () ^ "/../../assets/number-light.svg")
```

Comment:
Why don't we use `extension_path`?

Reply:
Propagating `extension_path` all the way to this function was very inconvenient, and looking at how other extensions are implemented, it seems to be the norm to use a relative path for the assets.

Reply:
Sure, we can use this approach instead. But then we should use it consistently everywhere. I'd also suggest to create a helper function `val asset : fname:string -> string` to encapsulate this detail.

Reply:
Done, I added an `asset` function in `Path`